metadata/metadata.go

```diff
@@ -22,6 +22,48 @@ import (
 	"strings"
 )
 
+// HasAnnotation checks if a given annotation exists
+func HasAnnotation(object v1.Object, key string) bool {
```

Comment:
I usually short functions alphabetically. Can you add the new ones after the GetLabelsWithPrefix one?

Reply:
Yes. Updated. Thanks